api/v1alpha1/opentelemetrycollectordistribution_types.go

```diff
@@ -0,0 +1,52 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kubebuilder:object:root=true
+
+// OpenTelemetryCollectorDistribution is the Schema for the opentelemetrycollectordistributions API
+type OpenTelemetryCollectorDistribution struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Image specifies the default image to use for this distribution. The image specified in the consuming resource should take precedence. Required.
```

Comment:
I'll rewrite this, the distribution one should take precedence.

Reply:
What about if the CLI flag is set, possibly this could be perceived as locking the operator into a single operand - so any CR that contains either the `image` or `distributionName` could fail?